src/main/java/catan/domain/model/game/GameBean.java

```diff
@@ -90,6 +90,9 @@
     @Column(name = "CURRENT_MOVE", unique = false, nullable = true)
     private Integer currentMove;
 
+    @Column(name = "OWNER_BIGGEST_ARMY", unique = false)
```

Comment:
Not clear what exact identifier is used (color, gameuserId or gameId, or moveorder) . maybe specify it in field name, or just add a comment